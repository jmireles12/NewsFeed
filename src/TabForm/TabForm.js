import React from 'react'
import './TabForm.css'

export default function TabForm(props) {
    const { className, ...otherProps } = props
    return (
        <form
        className={['Tab-form', className].join(' ')}
        action='#'
        {...otherProps}
    />
    )
}