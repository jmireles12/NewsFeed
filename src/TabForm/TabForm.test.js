import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import TabForm from './TabForm'

describe(`TabForm component`, () => {
    it('renders the complete TabForm', () => {
        const wrapper = shallow(<TabForm />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })
})