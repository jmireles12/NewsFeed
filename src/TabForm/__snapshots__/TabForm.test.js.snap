// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabForm component renders the complete TabForm 1`] = `
<form
  action="#"
  className="Tab-form "
/>
`;
